sklearn/tests/test_docstring_parameters.py

```diff
@@ -232,13 +232,10 @@ def test_fit_docstring_attributes(name, Estimator):
         with ignore_warnings(category=FutureWarning):
             assert hasattr(est, attr.name)
 
-    IGNORED = {'BayesianRidge', 'Birch', 'CCA', 'CategoricalNB',
-               'KernelCenterer',
-               'LarsCV', 'Lasso', 'LassoLarsIC',
-               'MiniBatchKMeans',
+    IGNORED = {'BayesianRidge', 'Birch', 'CCA',
+               'LarsCV', 'Lasso', 'LassoLarsCV', 'LassoLarsIC',
                'OrthogonalMatchingPursuit',
-               'PLSCanonical', 'PLSSVD',
-               'PassiveAggressiveClassifier'}
+               'PLSCanonical', 'PLSSVD'}
```

Comment:
I found that 3 other estimators pass the test. We forgot to remove them from the blacklist in the dedicated doc PRs